src/vcpkg/configuration.cpp

```diff
@@ -535,6 +537,66 @@ namespace
 
 namespace vcpkg
 {
+    static Optional<std::string> get_baseline_from_git_repo(const VcpkgPaths& paths,
+                                                            StringView url,
+                                                            LocalizedString& error)
```

Comment:
```suggestion
    static ExpectedL<std::string> get_baseline_from_git_repo(const VcpkgPaths& paths,
                                                            StringView url)
```

Reply:
ExpectedL has better behavior than Optional if you `value_or_exit` it, for example: printing the error text instead of "value was null"